### PR TITLE
Add Polycode 0.8.4

### PR DESCRIPTION
**Pull request by JaredSartin**

### CODE REVIEW

File: Casks/polycode.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'polycode' do
+  version '0.8.4'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/polycode.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/polycode.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  depends_on :macos => '>= :lion'
```
Comment (vitorgalvao) [inline Casks/polycode.rb]: These two lines should be the last ones.

File: Casks/polycode.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  depends_on :arch => :x86_64
+
+  url "http://polycode.org/download/content/PolycodeDarwin_#{version}.zip"
+  name 'Polycode'
+  homepage 'http://polycode.org/'
+  license :gratis
```
Comment (vitorgalvao) [inline Casks/polycode.rb]: `:mit`.

**Comment by JaredSartin** [comment]
@vitorgalvao I made those changes - Let me know if there are more.

**Comment by vitorgalvao** [comment]
Looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
Done in https://github.com/caskroom/homebrew-cask/commit/e65b021514e86daeeb26c462bb8c5e72fbac99af.

**Pull request closed without merging**